src/printer/smt2/smt2_printer.cpp

```diff
@@ -862,7 +862,8 @@ void Smt2Printer::toStream(std::ostream& out,
     out << ')';
     return;
   }
-  case kind::INST_PATTERN: break;
+  case kind::INST_PATTERN:
+  case kind::INST_NO_PATTERN: break;
```

Comment:
This case shouldn't really occur because of the `nc[0]` below, no?

Reply:
That's right, although the kind should appear here regardless.

Reply:
Should we maybe make it `Unreachable()` in that case?

Reply:
No, we still should be able to print it out internally.  It's just that it's main use case (as a child of forall) forgoes printing it.

Reply:
That's fair.